torch/_torch_docs.py

```diff
@@ -4138,6 +4138,45 @@ def merge_dicts(*dicts):
     tensor([ 0.8722, -0.7416,  0.2653, -0.1584])
 """.format(**single_dim_common))
 
+add_docstr(torch.amax,
+           r"""
+amax(input, dim, keepdim=False, *, out=None) -> Tensor
+
+Returns the maximum value of each row of the :attr:`input` tensor in the given
```

Comment:
Given that there are potentially multiple dimensions, it's `slice`, not `row`

Reply:
fixed